src/__tests__/App.test.tsx

```diff
@@ -1,16 +1,17 @@
+import { store } from '@store/store';
```

Comment:
Можно сделать индекс в папке и ссылаться на папку, а не на файл. Это поможет убрать лишнюю инфу в импорте

```suggestion
import { store } from '@store';
```